dotCMS/src/main/java/com/dotcms/rest/api/v1/index/ESIndexResource.java

```diff
@@ -67,392 +67,351 @@
-     * @param indexName
-     * @return
-     * @throws DotDataException
-     * @throws IOException
-     */
-    @Deprecated
-    public static File downloadIndex(String indexName) throws DotDataException, IOException {
-
-        if(indexName.equalsIgnoreCase("live") || indexName.equalsIgnoreCase("working")){
-            IndiciesInfo info=APILocator.getIndiciesAPI().loadIndicies();
-            if(indexName.equalsIgnoreCase("live")){
-                indexName = info.getLive();
-            }
-            if(indexName.equalsIgnoreCase("working")){
-                indexName = info.getWorking();
-            }
-        }
-
-        return APILocator.getESIndexAPI().backupIndex(indexName);
-
-    }
+        this.idxApi = APILocator.getContentletIndexAPI();
 
-    public static String create(String indexName, int shards, boolean live) throws DotIndexException, IOException {
-        if(indexName == null)
-            indexName=ContentletIndexAPIImpl.timestampFormatter.format(new Date());
-        indexName = (live) ? "live_" + indexName : "working_" + indexName;
+	}
 
-        APILocator.getContentletIndexAPI().createContentIndex(indexName, shards);
+    protected InitDataObject auth(
+                                  final HttpServletRequest request,
+                                  final HttpServletResponse response) {
 
-        return indexName;
+        return new WebResource
+                        .InitBuilder(request, response)
+                        .requiredRoles(Role.CMS_ADMINISTRATOR_ROLE)
+                        .requiredPortlet("maintenance")
+                        .init();
     }
+    
+    
+    @CloseDBIfOpened
+    @GET
+    @JSONP
+    @NoCache
+    @Path("/cluster")
+    @Produces({MediaType.APPLICATION_JSON})
+    public Response getClusterStats(@Context final HttpServletRequest request, @Context final HttpServletResponse response)
+                    throws DotDataException {
 
-    public static void activateIndex(String indexName) throws DotDataException {
-    	AdminLogger.log(ESIndexResource.class, "activateIndex", "Trying to activate index: " + indexName);
+        auth(request, response);
 
-        if(indexName.startsWith(SiteSearchAPI.ES_SITE_SEARCH_NAME)){
-            APILocator.getSiteSearchAPI().activateIndex(indexName);
-        }
-        else{
-            APILocator.getContentletIndexAPI().activateIndex(indexName);
-        }
 
-        AdminLogger.log(ESIndexResource.class, "activateIndex", "Index activated: " + indexName);
-    }
+        final ESIndexAPI esIndexAPI = new ESIndexAPI();
+        final ClusterStats clusterStats = esIndexAPI.getClusterStats();
 
-    public static void deactivateIndex(String indexName) throws DotDataException, IOException {
-    	AdminLogger.log(ESIndexResource.class, "deactivateIndex", "Trying to deactivate index: " + indexName);
+        Builder<String, Object> builder =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'builder' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)